src/server/items/items-routes.js

```diff
@@ -0,0 +1,33 @@
+import express from 'express';
+
+const router = express.Router();
+
+router.get('/', (req, res, next) => {
```

Comment:
We will! First we specify the REST with Swagger and then we will fill those functions